st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
```

Comment:
Right now, when st2client.js receives 200 code on POST, it throws an error (since it expects 201). We're catching the error in hubot-stackstorm (https://github.com/StackStorm/hubot-stackstorm/blob/master/scripts/stackstorm.js#L209-L217), checking status and constructing an object resembling expected response.

If you run a new hubot-stackstorm against an old st2, it would act as if no additional parameters are set. If you run an old hubot against a new st2, it would just output ypu a status code and a message body into the chat (https://github.com/StackStorm/hubot-stackstorm/blob/v0.2.5/scripts/stackstorm.js#L180)